src/wallet/asyncrpcoperation_common.cpp

```diff
@@ -146,18 +152,19 @@ void ThrowInputSelectionError(
                                     FormatMoney(dte.dustThreshold));
                         }
                     }))
-                    + (selector.TransparentCoinbasePolicy() != TransparentCoinbasePolicy::Disallow
-                       ? "" :
-                       "; note that coinbase outputs will not be selected if you specify "
-                       "ANY_TADDR, any transparent recipients are included, or if the "
-                       "`privacyPolicy` parameter is not set to `AllowRevealedSenders` or weaker")
-                    + (!isFromUa || strategy.AllowLinkingAccountAddresses() ? "." :
-                       ". (This transaction may require selecting transparent coins that were sent "
-                       "to multiple Unified Addresses, which is not enabled by default because "
-                       "it would create a public link between the transparent receivers of these "
-                       "addresses. THIS MAY AFFECT YOUR PRIVACY. Resubmit with the `privacyPolicy` "
-                       "parameter set to `AllowLinkingAccountAddresses` or weaker if you wish to "
-                       "allow this transaction to proceed anyway.)"));
+                    + (selector.TransparentCoinbasePolicy() == TransparentCoinbasePolicy::Disallow
+                       ? "; note that coinbase outputs will not be selected if you specify "
+                         "ANY_TADDR, any transparent recipients are included, or if the "
+                         "`privacyPolicy` parameter is not set to `AllowRevealedSenders` or weaker"
+                       : "")
```

Comment:
Inverting this conditional and simultaneously changing the indentation, in the same commit as the nominal change for this PR, made this harder to review. Please separate out formatting-only changes like this into separate commits in future.